packages/ui-drilldown/src/Drilldown/__examples__/Drilldown.examples.tsx

```diff
@@ -30,33 +30,227 @@ import { Drilldown } from '../index'
 import type { DrilldownProps } from '../props'
 
 export default {
-  // sectionProp: 'shape',
-  propValues: {},
-  filter: () => {
-    return false
+  sectionProp: 'trigger',
```

Comment:
I wouldn't use the trigger as a section prop, in fact I wouldn't use a section prop at all. I would rather add a few custom example cases where it has a `trigger`, and only test the `disabled`, `withArrow` and `defaultShow` props on them (maybe the `placement` too), because those seem to be the only ones relevant (all of the rest is handled inside the drilldown and will look the same as the flat version).